easybuild/tools/filetools.py

```diff
@@ -342,6 +342,8 @@ def search_file(paths, query, short=False, ignore_dirs=None, silent=False):
         raise EasyBuildError("search_file: ignore_dirs (%s) should be of type list, not %s",
                              ignore_dirs, type(ignore_dirs))
 
+    query = re.compile(query.lower())
```

Comment:
why not do it trough the regex support of options system? And why the `.lower()`?